bundles/statistics/statsgrid2016/components/IndicatorSelection.js

```diff
@@ -190,5 +242,54 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorSelection', function (
         var el = this.getElement();
         var indicSel = el.find('.stats-ind-selector');
         return indicSel;
+    },
+    /**
+     * @method  @public  getUnsupportedDatasets 
+     * @description returns a list of unsupported datasources for the currently selected regionset(s)
+     * @param regionsets regionsets
+     */
+    getUnsupportedDatasetsList: function ( regionsets ) {
+        if (regionsets === null) {
+            return;
+        }
+        regionsets = regionsets.map( function (id)  { return Number(id) } );
```

Comment:
The same number casting is done several times. Do it just once and pass the array of numbers around.